.aspect/workflows/terraform/workflows.tf

```diff
@@ -91,10 +98,23 @@ module "aspect_workflows" {
       min_runners               = 0
       queue                     = "aspect-default"
       resource_type             = "default"
-      scale_out_factor          = 2
       scaling_polling_frequency = 1 # check for queued jobs every 60s
       warming                   = true
     }
+    small = {
+      agent_idle_timeout_min = 1
+      gh_repo                = "aspect-build/rules_ts"
+      # Determine the workflow ID with:
+      # gh api -H "Accept: application/vnd.github+json" -H "X-GitHub-Api-Version: 2022-11-28" /repos/aspect-build/rules_ts/actions/workflows
+      # https://docs.github.com/en/rest/actions/workflows?apiVersion=2022-11-28#list-repository-workflows
+      gha_workflow_ids          = ["66360195"] # Aspect Workflows
+      max_runners               = 5
+      min_runners               = 0
+      queue                     = "aspect-small"
+      resource_type             = "small"
+      scaling_polling_frequency = 1 # check for queued jobs every 60s
+      warming                   = false
+    }
```

Comment:
Do we want a scale out factor on the small instances for the format/buildifier tasks?

Reply:
Decided to remove them in general on our staging repos to reduce compute costs